data/ignore_ids_safety_scan.json

```diff
@@ -36,7 +36,8 @@
                 "42098":"Can't upgrade TF version to 2.7.x in non 2.7 images.",
                 "42062":"Can't upgrade TF version to 2.7.x in non 2.7 images.",
                 "51159": "for cryptography until we have 39.0.0 release",
-                "44715":"False positive for numpy. Was fixed in 1.22.2"
+                "44715":"False positive for numpy. Was fixed in 1.22.2",
```

Comment:
As per the allowlist, looks like the issues were found for TF inference, TF inference eia, TF inference neuron, PT inference, PT inference EIA, PT inference-neuron, huggingface_tensorflow inference, huggingface_pytorch inference. Did we observe failures on all these images on any of the commits?

Reply:
Yes. Was found on EIA first, and then seen in the logs for the others when attempting to run through various pipeline sanity checks as wheel is in all the images. Still no update on pypi as of today.